pycaret/time_series/forecasting/oop.py

```diff
@@ -3972,6 +4011,10 @@ def check_stats(
             Dataframe with the test results
         """
 
+        if not self._setup_ran:
```

Comment:
Can you remove this for now (just for this method - `check_stats`). `check_stats` is being called from within `setup` before the `_setup_ran` flag is set, hence all unit tests are failing now.